src/components/drawer/Drawer.tsx

```diff
@@ -117,7 +118,13 @@ const attachToDy = (animation: Animated.Value, newValue: number) => (
   )(e, { dy: newValue })
 }
 
-const Drawer = ({ isOpen, children, onClose, isFullscreen }: DrawerProps) => {
+const Drawer = ({
+  isOpen,
+  children,
+  onClose,
+  isFullscreen,
+  isGestureSupported = true
```

Comment:
`isGestureSupported` sounds like it's referring to gesture support on the device. Maybe we can rename this to `isGestureEnabled` or `enableGestures`?

Reply:
sounds gouda